File: 2018/solutions/prac02.java
import java.io.*;

public class prac02 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader
                = new BufferedReader(new InputStreamReader(System.in));

        String line = null;
        while ((line = reader.readLine()) != null) {
            String words[] = line.trim().split("\\s+");
            String noun = words[0];
            String verb = words[1];
            System.out.println(noun + " is " + verb + " today!");
        }
    }
}

File: 2019/solutions/prac02.js
const readline = require('readline');

const readInterface = readline.createInterface({
    input: process.stdin,
    console: false
});

readInterface.on('line', function(line) {
    words = line.split(' ');
    console.log(words[0] + ' is ' + words[1] + ' today!');
});
